src/parser/view_transform.c

```diff
@@ -1669,7 +1669,7 @@ static PUSHABLE_TYPE
 mq_is_pushable_subquery (PARSER_CONTEXT * parser, PT_NODE * subquery, PT_NODE * mainquery, PT_NODE * class_spec,
 			 bool is_vclass, PT_NODE * order_by, PT_NODE * class_)
 {
-  PT_NODE *pred, *statement_spec = NULL, *orderby_for;
+  PT_NODE *pred, *statement_spec = NULL, *orderby_for, *select_list;
```

Comment:
How about initializing it ahead?

```suggestion
  PT_NODE *pred, *statement_spec = NULL, *orderby_for, *select_list = NULL;
```

Reply:
It is unnecessary in case of PT_SELECT.